docs/detections/rules-ui-create.asciidoc

```diff
@@ -136,6 +150,55 @@ per rule execution:
 ** _Field_: Leave blank
 ** _Threshold_: `10`
 
+* To create a new EQL rule, select _Event Correlation_ and then:
+.. Define which {es} indices the rule searches for alerts.
+.. Add an {ref}/eql-syntax.html[EQL statement] used to detect alerts.
++
+For example the following rule detects when `msxsl.exe` makes an outbound
+network connection:
+
+** _Index patterns_: `winlogbeat-*`
++
+Winlogbeat ship Windows events to {es-sec}.
```

Comment:
Is there a convention for visually distinguishing an idea from its description? I _think_ that's what these two lines are, but I (and presumably other readers) would benefit from some visual categorization/reinforcement here.

Reply:
@rylnd: Since I'm taking this over from Ben, I'm not sure which two ideas we're trying to distinguish. Are you hoping for a short flow of how the EQL query from winlogbeat moves from the process to the network?

Reply:
Hey @DonNateR ! Sorry, I was unclear there. I think the content is good, my question was centered around formatting:

<img width="680" alt="Creating_detection_rules___Elastic_Security_Solution__master____Elastic" src="https://user-images.githubusercontent.com/657252/96766316-7eeba680-13a0-11eb-862d-5a63b9990aa6.png">

I was having trouble parsing that text and I _think_ it's because the rule parameter and its accompanying description aren't visually distinct. If that's standard formatting for something like this, though, I'm all for consistency 👍

Reply:
@DonNateR aside from Ryland's comment around formatting, I think there is a typo in this line. I think it's supposed to say `Winlogbeat ships` as opposed to `Winlogbeat ship`.